docs/source/estimator_overview/estimator_overview.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js">
+  </script>
+
+  <style>
+    html {
+      font-family: "Lucida Sans Unicode", "Lucida Grande", Sans-Serif;
+    }
+
+    input {
+      background-image: url('https://www.w3schools.com/css/searchicon.png');
+      /* Add a search icon to input */
+      /*background-position: 10px 12px;*/
+      /* Position the search icon */
+      background-repeat: no-repeat;
+      /* Do not repeat the icon image */
+      width: 40%;
+      /* Full-width */
+      font-size: 16px;
+      /* Increase font-size */
+      padding: 8px 20px 8px 40px;
+      /* Add some padding */
+      border: 1px solid #ddd;
+      /* Add a grey border */
+      margin-bottom: 10px;
+      /* Add some space below the input */
+    }
+
+    table {
+      margin: 0 auto;
+      width: 100%;
+      clear: both;
+      border-spacing: 0;
+      border: 1px solid white;
+      overflow: auto;
+    }
+
+    th,
+    td {
+      border: 1px solid #ddd;
+      color: #6678b1;
+      text-align: left;
+      height: 50px;
+      margin: 0;
+      font-size: 12px;
+      /* Increase font-size */
+      outline: none;
+    }
+
+    thead th {
+      border-top: 1px solid #333131;
+      border-bottom: 1px solid #333131;
+      text-align: center;
+      font-size: 15px;
+      /* Increase font-size */
+      padding: 5px;
+      font-weight: bold;
+      outline: none;
+      color: #039 !important;
+    }
+
+    .dataframe tr:nth-child(even) {
+      background-color: #f2f2f2;
+    }
+  </style>
+</head>
+
+<body>
+<label for="myInput"></label><input type="text" id="myInput" placeholder="Search for names..">
+  <br>
+  <br>
+
+  <div
+          w3-include-html="estimator_overview_table.html"></div>
```

Comment:
```suggestion
          w3-include-html="../source/estimator_overview/estimator_overview_table.html"></div>
```

Reply:
Hopefully, it'll work.
```suggestion
          w3-include-html="estimator_overview_table.html"></div>
```

Reply:
Is it working for you locally? If so could you tell me, what you run in what sequence?

Reply:
Yes, it's working now fine. But it'll run through estimator_overview.rst file only.

Reply:
Okay, so what do you run to generate the docs?

Reply:
once you run a command using sphinx. Thanks.

Reply:
I guess `make docs` but sorry, I ran `sphinx-build source built`.

Reply:
When you run this, `open built/index.html` and navigate to the estimator overview, do you see the table? I still can't see it ... 🤔

Reply:
Thanks Markus. I see the table when I run that. Let's see at the meeting soon.